src/python/foglamp/translators/omf_translator.py

```diff
@@ -114,26 +88,29 @@
 # PI Server OMF reference - for detailed information
 # http://omf-docs.readthedocs.io/en/v1.0/Data_Msg_Sample.html#data-example
 
-_server_name = ""
+_event_loop = ""
+
+# Managed by initialize_plugin
 _relay_url = ""
 _producer_token = ""
+_channel_id = 1
+"""Channel Id for the OMF translator"""
+_omf_max_retry = 5
+_omf_retry_sleep_time = 1
+
 
 # The size of a block of readings to send in each transmission.
-_block_size = 50
+_block_size = 1000
```

Comment:
We need to be careful with the block size we choose. Too small and the performance will be poor, too large and we will build up a big in memory structure. I would suggest we make this a configuration options to allow for easy tuning in the short term. Longer term this may be variable on a per stream basis.